.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,44 @@
+# GitHub Issue for the Pull Request  
+Link to the relevant GitHub issue
+
+# GitHub Discussions Related to this Pull Request  
+List any GitHub discussion links related to this PR
+
+# Checklists
+
+Completing these checklists ensure that the contributor's PR meets our build server requirements on their first attempt.
+
+- [ ] I do not require assistance from NI to complete any of the following checks.
+- [ ] I [pushed the feature branch](https://github.com/ni/labview-icon-editor/wiki/git#featurebranch) associated with this GitHub issue from my fork.
+- [ ] I built a VI Package using the [Powershell build tool](https://github.com/ni/labview-icon-editor/wiki/automation#pwsh).
+- [ ] I installed the VI Package produced by the Powershell build tool and tested my change.
+- [ ] I tested my changes after [installing the VI package](https://github.com/ni/labview-icon-editor/wiki/test#localtesting).
+- [ ] NI has my contributor license agreement.
+
+# Summary of Changes
```

Comment:
If we assume that this comes through the Issue to PR workflow then making the submitter duplicate information that should be in the Issue and any discussion seems somewhat lazy on the part of reviewers (IMO)

I may ask for any information **not** captured in the issue - lets say the Issue discusses one implementation but the submitter opts for an alternative during development for instance

_but_

Otherwise the reviewer should get up-to-speed by reading the linked issue and review the PR from there.